src/main/ui/package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "isomorphic-fetch": "^2.2.1",
     "react": "^15.4.2",
+    "react-bootstrap": "^0.30.7",
```

Comment:
Eventually we should include this from CDN (which would be a change to index.ejs and to webpack.config.js), but we don't need to just yet.